version/src/lib.rs

```diff
@@ -67,8 +58,11 @@ impl Default for Version {
             major: env!("CARGO_PKG_VERSION_MAJOR").parse().unwrap(),
             minor: env!("CARGO_PKG_VERSION_MINOR").parse().unwrap(),
             patch: env!("CARGO_PKG_VERSION_PATCH").parse().unwrap(),
-            commit: compute_commit(option_env!("CI_COMMIT")),
+            // TODO: Is unwrap_or_default good enough?
+            commit: compute_commit(option_env!("CI_COMMIT")).unwrap_or_default(),
             feature_set,
+            // TODO: How to override this?
+            client: u16::try_from(ClientId::SolanaLabs).unwrap(),
```

Comment:
@mvines 

* How do you suggest to override `client` in here? I was thinking instead of `default` add a `fn new(client: ClientId) -> self` but that seems to be too intrusive all over the code-base.
  * an alternative is to just leave it hard-coded here with the expectation that other clients have to modify this line in their code.
* Also you mentions the `Option` around `commit` could be removed too. Did you mean use `unwrap_or_default` here or something else?

Reply:
@mvines do we want this change?

Reply:
sry for the delay.
I think it's ok to hard code `ClientId::SolanaLabs`. For example the first thing Jito would do it replace it with their own. Firedancer has a completely different implementation so no issue for them.

> Also you mentions the Option around commit could be removed too.
I was thinking that if CI_COMMIT isn't found then just use `0` instead of `None`. But on the other hand, `commit` doesn't really convey anything too useful. In the interest of reducing gossip egress I think we could remove it entirely

Reply:
> In the interest of reducing gossip egress I think we could remove it entirely

I think the impact on gossip wouldn't be much.
I am leaning to keep it since it can be helpful with debugging.

I updated the commit and removed `TODO`s.
Is this good to merge?